CONTRIBUTING.md

```diff
@@ -1,34 +1,46 @@
-## 🙏 Welcome
+## ✨ Welcome
 
 Thank you for your interest in Openverse! We're so excited to bring new contributors into the fold in WordPress, Openverse and FOSS in general.
 
 ## ❤️ Code of Conduct
 
 Please be aware that as a project of the WordPress Foundation, Openverse follows [the WordPress Etiquette](https://wordpress.org/about/etiquette/), a code of conduct that you should read and agree to abide by before contributing to WordPress projects. This applies to all Openverse respositories.
 
-## 😍 Keeping in Touch
+## 🎨 Design contributions
 
-For folks who'd like to be involved in our regular development conversations, please feel free to join us in the WordPress make chat (https://make.wordpress.org/chat/). You'll find us in the #openverse channel, of course! We have weekly community meetings on Tuesdays at 15:00 UTC (https://everytimezone.com/s/d1d42c7b). All are welcome! During the meetings we discuss ongoing work and assign new issues. Attendance is not mandatory to contribute, however. Any issues labeled as "good first issue" or "help wanted" in our repositories are all up for grabs. Just add a comment with `@WordPress/openverse-maintainers` on the issue with questions or requesting the issue be assigned to you when you're ready to work on it. We also regularly discuss Openverse development issues outside the meeting times (like debugging problems we're having while working through an issue).
+### How to contribute with design
 
-You can also keep in touch with [progress](https://github.com/orgs/WordPress/projects/3) and the latest updates to the project with our [WordPress contributor group](https://make.wordpress.org/openverse/).
+If you'd like to contribute to the design or front-end, feel free to propose a solution to an existing problem labeled with [Needs Design](https://github.com/WordPress/openverse-frontend/issues?q=is%3Aopen+is%3Aissue+label%3A%22needs+design%22), or share an idea you think it adds value to the Openverse goal.
```

Comment:
```suggestion
If you'd like to contribute to the design or front-end, feel free to propose a solution to an existing problem labeled with [Needs Design](https://github.com/WordPress/openverse-frontend/issues?q=is%3Aopen+is%3Aissue+label%3A%22needs+design%22), or share an idea if you think it meets Openverse's goals.
```

This might be a bit clearer.